Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM emberjs/website-dev:latest
+
+# Customizations for this project here.
+# e.g.: if you have dependencies in addition to what the base image provides:
+ADD Gemfile /src/Gemfile
+ADD Gemfile.lock /src/Gemfile.lock
+RUN bundle install
```

Comment:
A nice tip is to use `bundle check` before calling `bundle install`-> `bundle check || bundle install`